packages/styles/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@nullvoxpopuli/limber-styles",
+  "version": "0.0.0",
+  "description": "CSS used for Limber",
+  "main": "index.js",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "build": "yarn tailwindcss -o dist/tailwind.css",
+    "start": "yarn tailwindcss -o dist/tailwind.css --watch"
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.0",
+    "postcss": "^8.4.4",
```

Comment:
Do you still need Postcss?

Reply:
unsure, I was following the tailwind docs, and I don't know how things interact, but the build works, so... :shrug: